examples/state-change-events-demo.js

```diff
@@ -0,0 +1,36 @@
+import { constants } from '../dist/index.js';
```

Comment:
This example is helpful. 👍 Nit: Can we either add this JS to the HTML or make a directory for these two files to live in?